pkg/api/configsync/v1beta1/helmconfig.go

```diff
@@ -112,8 +112,7 @@ type ValuesFileRef struct {
 	// name represents the Object name. Required.
 	Name string `json:"name,omitempty"`
 
-	// valuesFile represents the object data key to read the value from.
-	// +kubebuilder:default:=values.yaml
+	// dataKey represents the object data key to read the value from. Default: `values.yaml`
```

Comment:
Suggest using plural given that it's called values.yaml

```suggestion
	// dataKey represents the object data key to read the values from. Default: `values.yaml`
```

Reply:
Fixed